configs/ib_config.yaml

```diff
@@ -0,0 +1,9 @@
+--- 
+generate:
+  - client
+  - types
+  - chi-server
```

Comment:
What do we want to generate? I tried models but it does not work for some reason, do you maybe know why?

Reply:
Just a client and types, that is all.